torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2084,6 +2084,17 @@ RegisterOperators reg2({
           return 0;
         }),
 
+    Operator(
+        "prim::divmod(int x, int y) -> (int, int)",
```

Comment:
please put this into `aten` namespace, everything that has schema should go to `aten` instead of `prim`, prim is reserved for unschematized operator, other ops like `abs` did it wrong and we will fix it in the future.